nimble/controller/src/ble_ll_adv.c

```diff
@@ -489,6 +505,52 @@ ble_ll_adv_pdu_make(uint8_t *dptr, void *pducb_arg, uint8_t *hdr_byte)
     return BLE_LL_EXT_ADV_HDR_LEN + ext_hdr_len;
 }
 
+#if (MYNEWT_VAL(BLE_LL_CFG_FEAT_LL_PERIODIC_ADV) == 1)
+static void
+ble_ll_adv_put_syncinfo(struct ble_ll_adv_sm *advsm, uint8_t *dptr)
+{
+    uint32_t offset;
+
+    /* TODO */
```

Comment:
TODO what?:)

Reply:
it is just intermediate code,  offset is always 0 here (ie not valid),  it is changed in next commit :)